llvm/include/llvm/SandboxIR/Tracker.h

```diff
@@ -213,6 +213,25 @@ class CallBrInstSetIndirectDest : public IRChangeBase {
 #endif
 };
 
+class SetVolatile : public IRChangeBase {
+  Instruction *Inst;
+  bool WasVolatile;
+  /// This could either be StoreInst or LoadInst
+  PointerUnion<StoreInst *, LoadInst *> InstUnion;
+
+public:
+  SetVolatile(Instruction *I, bool WasVolatile, Tracker &Tracker);
```

Comment:
No need for a `bool WasVolatile` argument, we can check `isVolatile()` within the constructor body.